ml/synthesis/src/consts.py

```diff
@@ -49,3 +49,6 @@ def _find_root_dir():
 MODELS_DIR = Path(_models_dir_env) if _models_dir_env else ML_SYNTHESIS_DIR / "models"
 
 ML_BACKEND_BASE_URL_FILEPATH = ".ml_backend_base_url"
+
+# high priority env var which overrides command provided in the config file
+NITTA_RUN_COMMAND_OVERRIDE = os.environ.get("NITTA_RUN_COMMAND", None)
```

Comment:
used env var name (`NITTA_RUN_COMMAND`) doesn't match
https://github.com/ryukzak/nitta/blob/ecd35f948ec03b9920b8f3fd06e73207bc8ef6ca/.github/workflows/ci.yml#L246-L247

Reply:
also it may be better to do it like
```python3
class EnvVarNames:
    MODELS_DIR = "NITTA_ML_SYNTHESIS_MODELS_DIR"
    NITTA_RUN_COMMAND = "NITTA_RUN_COMMAND"

...

NITTA_RUN_COMMAND_OVERRIDE = os.environ.get(EnvVarNames.NITTA_RUN_COMMAND, None)
```

it's consistent with the 
```python3
_models_dir_env = os.environ.get(EnvVarNames.MODELS_DIR)
```

Reply:
Don't fully understand the idea behind `EnvVarNames` class, but consistency is an argument. Thank you!

Reply:
The idea is quite boring, EnvVarNames.MODELS_DIR had to be reused in tests: 
https://github.com/ryukzak/nitta/blob/ffa93028f8f30ab34b84643dc658cbfe66c0dca9/ml/synthesis/src/tests/test_smoke.py#L80-L86

So I preferred to do it this way instead of hardcoding :)
Class is just for namespacing.

Regardless, I think it's good to have a single place where all the env var names (that python modules understand and use) are listed together, so they're not scattered across the code as literals. If there ever will be a need to generate docs for them, it'll make it simpler.